app_legacy/Helpers/database/search.php

```diff
@@ -130,7 +138,7 @@ function performSearch(int|array $searchType, string $searchQuery, int $offset,
         }
     }
 
-    if (count($articleTypes) > 0) {
```

Comment:
Using `count()` is expensive as it will actually count the items in the array. There are often better ways to check for "emptiness".